README.md

```diff
@@ -7,20 +7,18 @@ This website uses the Jekyll templating system and is hosted as a GitHub page ou
 * Ruby
 * `bundler` and [`jekyll`](https://jekyllrb.com):
 
-        gem install bundler jekyll
-
-    or locally:
-
-        gem install bundler jekyll --user
+gem install bundler jekyll
+or locally:
+ gem install bundler jekyll --user
 
 ## Setup
 1. While in cloned folder, run:
 
-        bundle install
+bundle install
```

Comment:
No indentation should change here... looks like maybe some text editor tool you were using might have messed with this automatically?